### DISCUSSION TITLE
AdGuardHome - Regex syntax (LatestVer)

### DISCUSSION THREAD

**Discussion by klisza1993** [AdguardTeam/AdGuardHome#5344] [General]
Hello. Can someone explain what is exacly syntax with REGEX in "Custom Filtering Rules"?
I was trying to block specific word in any domain. My examples look like this, tried many:
||/^(.+[_.-])?ajax.com/
||/^ajax/
ive also tried with $Important but no luck.
-----> Tested domain to see if filter is working <ajax.googleapis.com>
Have you got some advice?

**Comment by ammnt** [comment]
> Hello. Can someone explain what is exacly syntax with REGEX in "Custom Filtering Rules"?
> I was trying to block specific word in any domain. My examples look like this, tried many:
> ||/^(.+[_.-])?ajax.com/
> ||/^ajax/
> ive also tried with $Important but no luck.
> -----> Tested domain to see if filter is working <ajax.googleapis.com>
> Have you got some advice?

Try to remove pipes before the rules:
```
/^(.+[_.-])?ajax.com/
/^ajax/
```
